src/PKSim.Presentation/Views/Charts/PKSimChartFactory.cs

```diff
@@ -32,6 +33,22 @@ public SimulationTimeProfileChart CreateChartFor(IndividualSimulation individual
          return chart;
       }
 
+      public SimulationPredictedVsObservedChart CreatePredictedVsObservedChartFor(IndividualSimulation individualSimulation)
+      {
+         var chart = Create<SimulationPredictedVsObservedChart>();
```

Comment:
what about we create a method that takes a generic and does all of this? Feels like same code

Reply:
converted to generic